tests/testthat/test-rbabylon.R

```diff
@@ -30,56 +30,53 @@ test_that("bbi_dry_run() correctly returns object", {
   expect_identical(class(res), PROC_CLASS_LIST)
 })
 
-if (Sys.getenv("METWORX_VERSION") == "" && Sys.getenv("DRONE") != "true") {
-  skip("bbi_init only runs on Metworx or Drone")
-} else {
 
-  test_that("check_bbi_exe() correctly errors or finds paths", {
-    FAKE_BBI_PATH <- "/tmp/fake/bbi"
+skip_if_not_drone_or_metworx("bbi_init")
```

Comment:
all of these "changes" below are just un-indenting because I took this out of an `else` block.